### ISSUE TITLE
Lock schema version 3 not yet supported but in use by runtime

### ISSUE THREAD

**Issue by mfulton26** [denoland/vscode_deno#957]
**Describe the bug**

The VSCode extension does not recognize Deno lock schema version 3.

**To Reproduce**

1. create any **main.ts** with a **deno.json** and run it to produce a **deno.lock** file
2. open the **deno.lock** file and note the warnings

**Expected behavior**

The `deno.lock` document is valid against its schema and VSCode reports no warnings.

**Screenshots**

![image](https://github.com/denoland/vscode_deno/assets/5588957/4c18c649-9244-4fcc-9a04-779b7d525a46)

**deno.json**:

```json
{
  "tasks": {
    "test": "deno test"
  },
  "imports": {
    "std/": "https://deno.land/std@0.203.0/assert/",
    "fast-check": "npm:fast-check@^3.13.1"
  }
}
```

**deno.lock**:

```json
{
  "version": "3",
  "packages": {
    "specifiers": {
      "npm:fast-check@^3.13.1": "npm:fast-check@3.13.1"
    },
    "npm": {
      "fast-check@3.13.1": {
        "integrity": "sha512-Xp00tFuWd83i8rbG/4wU54qU+yINjQha7bXH2N4ARNTkyOimzHtUBJ5+htpdXk7RMaCOD/j2jxSjEt9u9ZPNeQ==",
        "dependencies": {
          "pure-rand": "pure-rand@6.0.4"
        }
      },
      "pure-rand@6.0.4": {
        "integrity": "sha512-LA0Y9kxMYv47GIPJy6MI84fqTd2HmYZI83W/kM/SkKfDlajnZYfmXFTxkbY+xSBPkLJxltMa9hIkmdc29eguMA==",
        "dependencies": {}
      }
    }
  },
  "remote": {
    "https://deno.land/std@0.203.0/assert/_constants.ts": "8a9da298c26750b28b326b297316cdde860bc237533b07e1337c021379e6b2a9",
    "https://deno.land/std@0.203.0/assert/_diff.ts": "1a3c044aedf77647d6cac86b798c6417603361b66b54c53331b312caeb447aea",
    "https://deno.land/std@0.203.0/assert/_format.ts": "a69126e8a469009adf4cf2a50af889aca364c349797e63174884a52ff75cf4c7",
    "https://deno.land/std@0.203.0/assert/assert_equals.ts": "d8ec8a22447fbaf2fc9d7c3ed2e66790fdb74beae3e482855d75782218d68227",
    "https://deno.land/std@0.203.0/assert/assertion_error.ts": "4d0bde9b374dfbcbe8ac23f54f567b77024fb67dbb1906a852d67fe050d42f56",
    "https://deno.land/std@0.203.0/assert/equal.ts": "9f1a46d5993966d2596c44e5858eec821859b45f783a5ee2f7a695dfc12d8ece",
    "https://deno.land/std@0.203.0/fmt/colors.ts": "c51c4642678eb690dcf5ffee5918b675bf01a33fba82acf303701ae1a4f8c8d9"
  }
}
```

warnings:

> ⚠️ Value is not accepted. Valid values: "2".
> ⚠️ Incorrect type. Expected "string".

**Versions**

vscode: 1.82.2 deno: 1.37.1 extension: v3.25.1

**Comment by nayeemrmn** [comment]
Fixed by #965.

**Comment by babakfp** [comment]
@nayeemrmn

```
Value is not accepted. Valid values: "3".(1)
```

![Screenshot 2023-11-09 192700](https://github.com/denoland/vscode_deno/assets/44144724/98abbfa2-3549-40ae-814d-0647ad2c2881)

Hi. I don't really understand the problem, however, the issue that I'm having in this screenshot looks very similar to the same issue that is discussed here. I thought maybe you can take a look at it and...
